crates/ruff_linter/src/rules/pycodestyle/rules/doc_line_too_long.rs

```diff
@@ -64,7 +64,7 @@ impl Violation for DocLineTooLong {
     #[derive_message_formats]
     fn message(&self) -> String {
         let DocLineTooLong(width, limit) = self;
-        format!("Doc line too long ({width} > {limit} characters)")
+        format!("Doc line too long ({width} > {limit} width)")
```

Comment:
I don't know if `width` makes sense at the end here because it's not a unit. E.g., `(80 > 79 width)` seems incorrect. Perhaps just `(80 > 79)`? Could `(80 > 79 characters)` be arguably fine to keep as-is, since the length is 79 single-width characters?

Reply:
or "width 80 > max width 79" or "width 80 > 79" or "width of 80 > 79"

Reply:
Isn't it a unit? I first used Unicode widths but it felt lengthy. 

I already found the existing one confusing because 88 > 80 characters compares unitless 88 with 80 characters. I like @zanieb's suggestion. Or I would write it as 88 > 80 [width]. I think that's the notation we used in university to denote the unit